### DISCUSSION TITLE
Check-in List exclude products

### DISCUSSION THREAD

**Discussion by astrocbxy** [pretix/pretix#2042] [Ideas]
At the moment you can have all products included in your check_in list or limit it to certain products. There is no possibility (I know of) to exclude certain products but auto include every new created product. I think it should be possible to have exclude_product field in the CheckinList model and filter out which products should be excluded.

**Comment by raphaelm** [comment]
Sorry, but no. This adds just too much complexity both to the configuration interface as well was to the code of pretix and the check-in apps.

**Reply by raphaelm** [reply]
An alternative I'd consider would be copying the association with check-in lists when copying a product. Wouldn't that solve the same problem in most cases?

**Reply by astrocbxy** [reply]
Yes it think it would.
